Directory.Packages.props

```diff
@@ -12,10 +12,11 @@
     <PackageVersion Include="Microsoft.CodeAnalysis.RulesetToEditorconfigConverter" Version="3.3.3" PrivateAssets="All" />
     <PackageVersion Include="Microsoft.CSharp" Version="4.5.0" />
     <PackageVersion Include="Microsoft.Identity.Client.NativeInterop" Version="$(MSALRuntimeNativeInteropVersion)" IncludeAssets="all" />
-    <PackageVersion Include="Microsoft.IdentityModel.Abstractions" Version="6.22.0" />
+    <PackageVersion Include="Microsoft.IdentityModel.Abstractions" Version="6.35.0" />
```

Comment:
why not v 7.2.0  ?

Reply:
There are some dependency issues if we upgrade to 7.x

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/4480#pullrequestreview-1793613904